### PR TITLE
Add locking queries to the reference guide

### PR DESCRIPTION
**Pull request by antonio2368**

### CODE REVIEW

File: products/memgraph/upcoming/reference_guide/configuration.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ Each configuration setting is in the form: `--setting-name=value`.
 | --also-log-to-stderr=false | log messages go to stderr in addition to logfiles | `[bool]` |
 | --data-directory=/var/lib/memgraph | Path to directory in which to save all permanent data. | `[string]` |
 | --log-file=/var/log/memgraph/memgraph.log | Path to where the log should be stored. | `[string]` |
+| --log-level | Minimum log level. Allowed values: TRACE, DEBUG, INFO, WARNING, ERROR, CRITICAL | `[string]` |
```
Comment (antonio2368) [inline products/memgraph/upcoming/reference_guide/configuration.md:59]: @g-despot Is this generated or do we manually add this?

**Review by gitbuda — APPROVED** [review]
Seems good!

File: products/memgraph/upcoming/reference_guide/making-a-backup.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+## Making a backup 
+
+While running, Memgraph generates couple of different files in its data
+directory. The directory includes multiple different subdirectories, one of
+them being the storage directory which contains the durablity files. In that
```
Comment (the-joksim) [inline products/memgraph/upcoming/reference_guide/making-a-backup.md:5]: durability

File: products/memgraph/upcoming/reference_guide/making-a-backup.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+## Making a backup 
+
+While running, Memgraph generates couple of different files in its data
+directory. The directory includes multiple different subdirectories, one of
+them being the storage directory which contains the durablity files. In that 
+directory Memgraph generates periodically snapshots and WAL files which
```
Comment (the-joksim) [inline products/memgraph/upcoming/reference_guide/making-a-backup.md:6]: periodically generates

File: products/memgraph/upcoming/reference_guide/making-a-backup.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+contain Memgrap's data in a recoverable format. 
+
+Making a backup of a Memgraph instance would consist of simply copying the
+data directory. This is not possible to do without additional help because
+the durabiltiy files are being deleted after some event is triggered
```
Comment (the-joksim) [inline products/memgraph/upcoming/reference_guide/making-a-backup.md:11]: are being -> can be

after some -> when an

**Review by the-joksim — COMMENTED** [review]
Well done! Just a couple of minor mistakes to correct.

File: products/memgraph/upcoming/reference_guide/configuration.md (hunk 1 above)
Comment (g-despot) [reply products/memgraph/upcoming/reference_guide/configuration.md:59]: Still being done manually.

**Review by g-despot — APPROVED** [review]

**Pull request merged**